docs/src/filters/local_rate_limit.md

```diff
@@ -20,7 +21,7 @@ static:
     - name: quilkin.extensions.filters.local_rate_limit.v1alpha1.LocalRateLimit
       config:
         max_packets: 1000
-        period: 500ms
+        period: 1s
```

Comment:
Yeah those work, the library we use parses things like that properly but we should probably call them out explicitly in the docs. Though now I'm more thinking this needs to be a plain int (number of seconds) rather than a duration type since we now round off to seconds, so folks aren't tempted to write e.g `period: 1300ms` and get surprised why the behavior is equivalent to `period: 1s`

Reply:
Updated this field to now use an int seconds